recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
-{% set name = "sk-video" %}
-{% set version = "1.1.10" %}
-{% set sha256 = "fb7125cfa2b31253dcf181d0bf285edd6bb25ffebf61dbd8ba11fadb783609a1" %}
+{% set name = "scikit-video" %}
```

Comment:
Just this is enough. We don't need to rename the feedstock to get the packaged renamed.

However, if you want to solve that inconsistency, you can re-submit to staged-recipes the recipe and we can archive this feedstock.

Reply:
so there's no way to rename the feedstock? can't the admins simply change the feedstock name? Or are there other subtle issues in doing this?

Reply:
ok, thanks @ocefpaf, my OCD prefers the "clean" way 😀.... I will resubmit then to staged-recipies and ask for deletion of this feedstock in the PR.

Reply:
Cool. Feel free to ping me over at staged-recipes.

PS: we don't delete the feedstock, we only archive them so they stop accepting PRs and issues.